### ISSUE TITLE
Using Variables that exist in the YAML or JSON files inside your markdown.

### ISSUE THREAD

**Issue by andyfensham** [nuxt/content#88]
Is it possible to use the variables that comes from the YAML def or the JSON files or CSV files inside your markdown on a specific page.

In an index page I can go and fetch all the content of a specific folder inside the contents directory.  I can list the content and then route to a specific page for e.g. markdown.  But I want to use the same variables that was available when I received the content through this.$content inside the actual markdown page instead of manually typing it.

**Comment by benjamincanac** [comment]
Hey @andyfensham,

When using `nuxt-content` component, all the document variables are injected inside the markdown, so you can do for example:

```md
---
title: Introduction
src: https://content.nuxtjs.org/icon.png
---

<img :src="src"></img>
```

**Comment by crazymind** [comment]
@benjamincanac 

What would be the solution here if you try display a variable out in a html attribute?
This for an example **won't** work:
```lisp
---
title: Introduction
src: https://content.nuxtjs.org/icon.png
---
 
<h1>{{ title }}</h1>
```

**Comment by nwittwer** [comment]
@crazymind As far as I can tell, there isn't a solution for interpolating these frontmatter YML variables within the document in Nuxt Content v1, i.e. `{{ title }}` becomes `Introduction` once compiled.

It does look like this is coming to Nuxt Content v2, along with a new [MDC (Markdown Components)](https://content.nuxtjs.org/guide/writing/mdc/) format. Looks like the PR has been merged, but is not yet released as of v2.0.1: https://github.com/nuxt/content/pull/1266

That being said, it would be really great to see the frontmatter variable interpolation in Nuxt Content v1 as well. Nuxt v2 + Content v2 are not stable and documented enough from what I've tried.
